.github/workflows/test-e2e.yml

```diff
@@ -4,43 +4,39 @@ on:
   push:
     branches:
       - master
-    paths-ignore:
-      - '.github/workflows/build.yml'
-      - '.github/workflows/codeql-analysis.yml'
-      - '.github/workflows/fill-gh-draft-release.yml'
-      - '.github/workflows/generate-documentation.yml'
-      - '.github/workflows/release.yml'
-      - '.github/workflows/upload-demo-archive.yml'
-      - '.gitignore'
-      - 'docs/**'
-      - '*.md'
-      - LICENSE
+    paths:
```

Comment:
Like @aibcmars said previosly, we should use:
`on: [push, pull_request]`

It will permit to avoid paths duplication.